cpp/scenario/gazebo/include/scenario/gazebo/utils.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Istituto Italiano di Tecnologia (IIT)
+ * All rights reserved.
+ *
+ * This project is dual licensed under LGPL v2.1+ or Apachi License.
```

Comment:
```suggestion
 * This project is dual licensed under LGPL v2.1+ or Apache License.
```

Reply:
Woops :) Good catch, I didn't notice it.

Reply:
I think I've updated all the files